packages/graphql/src/index.ts

```diff
@@ -0,0 +1,17001 @@
+/* THIS FILE IS AUTO-GENERATED, DO NOT EDIT */
```

Comment:
I get a big diff in this file after building.  It want's to take a bunch of trailing commas off.  Is there a settings difference between your workspace and mine?

Reply:
This file should not be merged. I feel like I may have made a mistake but I'm not sure what 😅 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/debt_5072_move-common-to-packages/packages/graphql/.gitignore